spanner_orm/admin/column.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 """Model for interacting with Spanner column schema table."""
 
+import typing
```

Comment:
it's a bit odd to see these two lines together but this does seem to be right way to do it

Reply:
Yup. Between https://www.python.org/dev/peps/pep-0585/ and `typing.cast` / `typing.overload` being generally rare, it shouldn't happen much in the future at least.